### ISSUE TITLE
 NETStandard 2.0

### ISSUE THREAD

**Issue by pazicb** [dbolin/Apex.Serialization#117]
Hello,
would it be possible for version 2.x to still support .NETStandard 2.0?

**Comment by dbolin** [comment]
Support for .NET Standard was deliberately dropped from version 2.x for a couple reasons.  First, .NET Standard doesn't really have much a future, other than if you need a library that supports both .NET framework and the newer .NET versions ([Microsoft later made this clear here](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/)).  Second, supporting .NET framework was quite a bit of operational and maintenance work, as I had to write special code that was conditionally compiled, resulting in differences in performance of the library (targeting .NET framework is slower) and having to run sets of tests (sometimes with different assertions) against both targets.

So basically I dropped that target since it's a bunch more work, and I expect most persons developing new projects that would use a library like this would (or should) not be targeting .NET framework. Currently, I don't have any plans to bring that target back.

**Comment by vvolkgang** [comment]
@dbolin Can understand that, unfortunately it cuts mobile (xamarin) support in the process, at least until .NET 6. Was looking forward to using Apex here: https://github.com/vvolkgang/CompressionChallenge

**Comment by dbolin** [comment]
If you're using Xamarin for iOS, it probably won't work out too well, since JIT compilation isn't supported there and Apex requires JIT compilation to function - I think Android supports JIT but not sure.  I'll update the readme.
